src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -393,23 +400,25 @@ private void bulkPutLocks(RoutingContext ctx, JsonArray locks) {
 
     private void getQueueItemsCount(RoutingContext ctx) {
         decodedQueueNameOrRespondWithBadRequest(ctx, lastPart(ctx.request().path())).ifPresent(queue ->
-                eventBus.request(redisquesAddress, buildGetQueueItemsCountOperation(queue), (Handler<AsyncResult<Message<JsonObject>>>) reply -> {
-                    if (reply.failed()) {
-                        log.warn("Failed to getQueueItemsCount", reply.cause());
-                        // Continue, only to keep backward compatibility.
-                    }
-                    if (reply.succeeded() && OK.equals(reply.result().body().getString(STATUS))) {
-                        JsonObject result = new JsonObject();
-                        result.put("count", reply.result().body().getLong(VALUE));
-                        jsonResponse(ctx.response(), result);
-                    } else {
-                        respondWith(StatusCode.INTERNAL_SERVER_ERROR, "Error gathering count of active queue items", ctx.request());
-                    }
-                }));
+                eventBus.request(redisquesAddress, buildGetQueueItemsCountOperation(queue),
+                (Handler<AsyncResult<Message<JsonObject>>>) reply -> {
+            if (reply.failed()) {
```

Comment:
So I propose to apply to the whole stuff the GoogleCodeFormatter and then we don't have to discuss anymore about this at any time :-)  The default formatter of IntelliJ is not really helpful here and it would not match the whole rest of the code anymore.  Will try to manually optimize.